### PR TITLE
Contact group members changes (SHOOP-2606)

### PR DESCRIPTION
**Pull request by tulimaki**
Admin: Hide group members in contact group edit view
Admin: Filter contacts with contact group in contact list

### CODE REVIEW

**Comment by tulimaki** [comment]
retest this please

**Comment by shawnadelic** [comment]
Looks good.

File: shoop/admin/templates/shoop/admin/contact_groups/_edit_members_form.jinja [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% from "shoop/admin/macros/general.jinja" import content_block %}
 {% set members_formset = form["members"] %}
 
-{% call content_block(_("Members"), "fa-users") %}
+{% call content_block(_("Add Members"), "fa-users") %}
```
Comment (shawnadelic) [inline shoop/admin/templates/shoop/admin/contact_groups/_edit_members_form.jinja]: Need to remake translations here

Comment (tulimaki) [inline shoop/admin/templates/shoop/admin/contact_groups/_edit_members_form.jinja]: Fixed!

**Pull request merged**